### ISSUE TITLE
Creating a model without splitting the dataset into a training and a test set

### ISSUE THREAD

**Issue by papadako** [guoguibing/librec#177]
Hi, I am not sure if this is the appropriate place to report this, but I didn't find a mailing-list. 

I have been wondering which is the appropriate way to create a model for a recommendation algorithm in such a way that librec does not split this to a training set and a test set. What I am doing (which I do not believe is the appropriate way) is to add the follow parameters to my configuration:
`data.input.path=u1.base

data.model.splitter=testset

data.testset.path=uNone.test`

where uNone.test is an empty file.

Is this correct? I guess it would be really helpful to have an option like data.model.splitter=None
